src/components/WhoWeAre/index.css

```diff
@@ -0,0 +1,58 @@
+@import url("https://fonts.googleapis.com/css?family=Acme&display=swap");
+
+.whoWeAre {
+  font-family: "Acme";
+}
+.targetText {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 1.8em;
+  margin: 1.8em;
+  width: 100%;
+  line-height: 170%;
+  text-align: center;
+  height: auto;
+}
+
+.targetHeader {
+  font-size: 1.8em;
+  color: #03c4a1;
+}
+
+.targetParagraph {
+  padding: 1.8em;
+  font-size: 1.1em;
+  color: #726a95;
+}
+
+img {
+  width: 100%;
+  height: auto;
+  padding: 10%;
+  margin: 5%;
+}
+
+.wrapper {
+  width: 100%;
+  height: 100%;
+  margin: 0 auto;
+}
+.innerVideo {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background: #03c4a1;
+  padding: 2%;
+}
+
+h1 {
```

Comment:
Can you give this h1 a className, because it affects all the h1 in the websites this way, and maybe give it another tag instead of h1 cause h1 should be used only one time a website